pages/certificat/[idCertificat].js

```diff
@@ -9,6 +9,21 @@ import Section from '@/components/section'
 const {NEXT_PUBLIC_API_BAN_URL} = getConfig().publicRuntimeConfig
 
 function Certificat({certificat = {}, id = '1234'}) {
+  const formatDate = dateString => {
+    const date = new Date(dateString)
+    const options = {
+      year: 'numeric',
+      month: 'long',
+      day: 'numeric',
+      hour: '2-digit',
+      minute: '2-digit',
+      second: '2-digit',
+      timeZone: 'UTC',
+      timeZoneName: 'short', // This will show 'UTC'
```

Comment:
C'est juste un détail, mais on veux vraiment du UTC ?
(Pour l'affichage, on peut mettre l'heure de Paris avec la valeur 'Europe/Paris'.)

```suggestion
      timeZone: 'Europe/Paris',
      timeZoneName: 'long', // This will show 'heure d’été d’Europe centrale'
```